### PR TITLE
DOCS-adding fixes for image assurance docs

### PR DESCRIPTION
**Pull request by penkeysuresh**
Product Version(s): Calico Cloud

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Review by girishshankaran — COMMENTED** [review]
LGTM. Please make these changes in the 3.17 folder as well.

**Review by bmckercher123 — APPROVED** [review]
Approved. Changes here reflect 3.18 so copy to 3.17 and 3.16 (current).

File: calico-cloud_versioned_docs/version-3.17/image-assurance/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (stevegaossou) [inline calico-cloud_versioned_docs/version-3.17/image-assurance/overview.mdx:1]: @penkeysuresh could you clarify why we need v3.17 docs? That hasn't been released yet.

Comment (penkeysuresh) [reply calico-cloud_versioned_docs/version-3.17/image-assurance/overview.mdx:1]: I was following the request, could you clarify this @bmckercher123 ?

**Review by stevegaossou — APPROVED** [review]
I'll pre-emptively approve. I defer to the docs team in terms of the release versions.

**Pull request merged**